### ISSUE TITLE
`meshopt` doesn't find meshoptimizer npm package

### ISSUE THREAD

**Issue by hybridherbst** [donmccurdy/glTF-Transform#615]
**Describe the bug**
I'm trying to use `gltf-transform meshopt` via CLI.
It errors with
```
error: meshopt: encoder dependency required — install "meshoptimizer".
```
which I'm happy to do
```
npm install -g meshoptimizer
```
but the error persists. Is there a way to use the CLI for `gltf-transform meshopt`? I kind of remember using that in the past.
The code just mentions an encoder has to be passed in, which I think I can't do for the CLI:
- https://github.com/donmccurdy/glTF-Transform/blob/4ad3f0e18defd39d777728af8c0b97eb9daf2831/packages/extensions/src/ext-meshopt-compression/meshopt-compression.ts#L58

**To Reproduce**
1. Use the CLI to compress any glb with `gltf-transform meshopt MyFile.glb MyFile.meshopt.glb`

**Expected behavior**
Some way to use `gltf-transform meshopt` via CLI

**Versions:**
 - Version: 2.1.7
 - Environment: CLI

**Additional context**
I think this might be a regression

**Comment by donmccurdy** [comment]
Does look like a regression, thanks! Should be fixed by #617.

**Comment by donmccurdy** [comment]
Published to v2.1.8, thanks for reporting the issue! 🙏🏻

**Comment by hybridherbst** [comment]
Wow, that was fast - thanks!
Also happy to see that `gltf-transform meshopt` doesn't have this issue :)
- https://github.com/zeux/meshoptimizer/issues/433
